test/data/failing/user-tests/build.gradle

```diff
@@ -0,0 +1,139 @@
+
```

Comment:
why use gradle file for the user tests? better to use maven like the application itself

Reply:
it is easy for me to implement gradle